README.md

```diff
@@ -115,11 +115,17 @@ provider.
 Set required environment variables based as per your infrastructure settings
 
 ```sh
+$ # API token provided from CSP
```

Comment:
What permissions should be in the token, can you explain here?

Reply:
Sure. Fixing in next commit.